packages/nx/src/plugins/package-json-workspaces/create-nodes.ts

```diff
@@ -84,12 +85,23 @@ export function createNodeFromPackageJson(pkgJsonPath: string, root: string) {
 
 export function buildProjectConfigurationFromPackageJson(
   packageJson: PackageJson,
+  root: string,
   path: string,
   nxJson: NxJsonConfiguration
 ): ProjectConfiguration & { name: string } {
   const normalizedPath = path.split('\\').join('/');
   const directory = dirname(normalizedPath);
 
+  const siblingProjectJson = tryReadJson<ProjectConfiguration>(
+    join(root, directory, 'project.json')
+  );
```

Comment:
I was confused by what is the `path`, and what is the `root` here.... 

Would this be better if we optionally passed in a parent `project.json`?